nhsblocks.php

```diff
@@ -156,36 +156,41 @@ function nhsblocks_register_dynamic_blocks() {
 		return;
 	}
 
-	$blocks = [
+	$blocks = array(
 		'nhsblocks/contentslistpage',
-	];
+	);
 
 	foreach ( $blocks as $block ) {
 
 		register_block_type(
 			$block,
-			[
-				// https://github.com/WordPress/gutenberg/issues/4671
+			array(
```

Comment:
Why has that changed to array? Shall we use [] as it was - recommended and fully supported in PHP >= 5.4